### PR TITLE
New new design signup

### PR DESCRIPTION
**Pull request by emmabckstrm**

### CODE REVIEW

File: app/assets/main/components/button.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
     border rounded border-current
     bg-transparent
     whitespace-no-wrap
-    text-primary
+    text-primary text-center
```
Comment (kallenilver) [inline app/assets/main/components/button.scss:8]: Newbie-question, I take it that it's our coding standard to have class names that concern the same thing (like text in this case) on the same row?

File: app/assets/main/components/callout.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .callout {
-  @apply bg-white rounded-lg shadow-lg p-4;
+  @apply bg-white rounded-lg shadow-lg p-4
+  border border-gray-tint-2;
```
Comment (kallenilver) [inline app/assets/main/components/callout.scss:3]: And that indentation after `@apply `is only needed if `@apply` is on an own row?

File: app/assets/main/components/button.scss (hunk 1 above)
Comment (stefanlindbohm) [reply app/assets/main/components/button.scss:8]: Yep. We haven't codified it but this is what we started doing in practice :)

File: app/assets/main/components/callout.scss (hunk 2 above)
Comment (stefanlindbohm) [reply app/assets/main/components/callout.scss:3]: Nah, this one I think is wrong. Always indent!

File: app/views/devise/registrations/new_new.html.erb [hunk 3]
```diff
@@ -1,162 +1,200 @@
...
-        </button>
-        <p class="text-orange-shade-1 ml-4" data-target="registration-form.errorMessage"></p>
+
       </div>
     </div>
+  </div>
+</section>
+
+<div class="bg-blue-tint-1 relative">
+  <%= image_tag webpack_asset_path('images/illustrations/forest_and_city.png'), alt: 'Forest and city illustration', class: 't:max-w-3/4 mx-auto' %>
```
Comment (stefanlindbohm) [inline app/views/devise/registrations/new_new.html.erb:171]: I think this should be limited to its size as in the design file, i.e. `class="w-3/4 mx-auto" style="max-width: 812px"`. It gets fuzzy and unnecessarily big on bigger screens now.

File: app/views/devise/registrations/new_new.html.erb [hunk 4]
```diff
@@ -1,162 +1,200 @@
...
+  <%= image_tag webpack_asset_path('images/illustrations/horizon_curve.svg'), alt: 'Curve illustration', class: 'w-full absolute bottom-0 -mb-1' %>
```
Comment (stefanlindbohm) [inline app/views/devise/registrations/new_new.html.erb:172]: I like the touch of adding `alt`s to images, but these are purely decoration so maybe these instead be hidden from screen readers? The current texts would be slightly confusing to hear, right?

**Pull request merged**